app/lib/common/widgets/html_editor/components/mention_item.dart

```diff
@@ -8,27 +8,19 @@ class MentionItem extends StatelessWidget {
     required this.mentionId,
     required this.displayName,
     required this.avatarOptions,
-    required this.isSelected,
     required this.onTap,
   });
 
   final String mentionId;
   final String? displayName;
   final AvatarOptions avatarOptions;
-  final bool isSelected;
 
   final VoidCallback onTap;
 
   @override
   Widget build(BuildContext context) {
-    final hasKeyboard =
```

Comment:
disabling the keyboard navigation selection (desktop) for now as it needs better solution for toggling between editor and mention list selected item focus in a separate PR.